src/chipeitlib/src/main/java/es/chipeit/lib/core/Registers.kt

```diff
@@ -4,7 +4,7 @@ import es.chipeit.lib.interfaces.IMemory
 import es.chipeit.lib.interfaces.IRegisters
 
 internal class Registers(override val v: IMemory<Byte>) : IRegisters {
-    override var i: Short = 0x00
+    override var i: Int = 0x00
```

Comment:
Register `I` is used to address memory, so it should be initialized to `= 0x000`. I'm surprised you left this one behind, taking into account that your review was a huge PITA with this issue.